VERSION

```diff
@@ -1 +1 @@
-0.6.14-dev
+0.6.14
```

Comment:
I think we've decided not to do a release in the same PR as other changes... @ray-milkey

Reply:
Done